documentation/hub/automation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: documentation
+menu: hub_automation
+subtitle: Automating Pre-flight Checks with Flyway Hub
+redirect_from: /documentation/hub-automation
+---
+
+# Automating Pre-flight Checks
+
+If your Pre-flight checks aren't run before every database deployment, their value diminishes and issues can slip through the cracks. We've made automating Pre-flight checks really easy, especially if you are able to use GitHub actions.
```

Comment:
Why do we capitalise 'Pre-flight' everywhere? Looks weird to me. I'd go with just 'pre-flight'.

Reply:
Done